lib/view.dart

```diff
@@ -104,9 +104,40 @@ void clearContentTab() {
   content.children.clear();
 }
 
-void renderAnalyseTab() {
-  clearContentTab();
-  content.append(html.DivElement()..innerText = 'Analyse');
+void renderAnalysisTabRadio(List<String> labels) {
+  var wrapper = html.DivElement()..classes = ['row'];
+  var labelCol = html.DivElement()
+    ..classes = ['col-lg-2', 'col-md-3', 'col-sm-12', 'col-xs-12']
+    ..innerText = 'Analyse';
+  var radioCol = html.DivElement()
+    ..classes = ['col-lg-10', 'col-md-9', 'col-sm-12', 'col-xs-12'];
+
+  for (var i = 0; i < labels.length; ++i) {
+    var radioWrapper = html.DivElement()
+      ..classes = ['form-check', 'form-check-inline'];
+    var radioOption = html.InputElement()
+      ..type = 'radio'
+      ..name = 'analyse-tab-options'
+      ..id = 'analyse-tab-options-${labels[i]}'
```

Comment:
would it be possible for the label to contain whitespace? or are we restricting it so that it doesn't?

you could probably just directly use the index in the id:
```suggestion
      ..id = 'analyse-tab-options-$i'
```

Reply:
Yes it might have white spaces, didn't think of it at all. Fixed